docs/UserGuide.md

```diff
@@ -145,22 +145,30 @@ Help is also shown if you enter an incorrect command e.g. `abcd`
 
 Adds a floating task to TaskTracker.
 
-    add "FLOATING_TASK_NAME" [PRIORITY]
+    task "FLOATING_TASK_NAME" [PRIORITY]
```

Comment:
An alternative sololution is we force the user to include that and time, but we provide shortcuts , which would still be conveinient after all. This means we dont assume and excluding any of the fields: start time, start date, end time, end date,  must all be included in the command format. for example,
floating task has to have start tie and start date, but can be written as 3pm tdy instead fo 3pm 10oct2016
events can be written as 3pm tdy 6pm tmr, it is still convienient